SPECS/dracut/dracut.spec

```diff
@@ -1,48 +1,62 @@
-%define dracutlibdir %{_libdir}/dracut
-%define _unitdir %{_libdir}/systemd/system
+%define dracutlibdir        %{_libdir}/%{name}
+%global __requires_exclude  pkg-config
 
 Summary:        dracut to create initramfs
 Name:           dracut
-Version:        055
-Release:        7%{?dist}
+Version:        059
+Release:        12%{?dist}
 # The entire source code is GPLv2+
 # except install/* which is LGPLv2+
 License:        GPLv2+ AND LGPLv2+
 Vendor:         Microsoft Corporation
 Distribution:   Azure Linux
 Group:          System Environment/Base
-URL:            https://dracut.wiki.kernel.org/
-Source0:        http://www.kernel.org/pub/linux/utils/boot/dracut/%{name}-%{version}.tar.xz
-Source1:        https://www.gnu.org/licenses/lgpl-2.1.txt
-Source2:        mkinitrd
-Source3:        megaraid.conf
-Source4:        20overlayfs/module-setup.sh
-Source5:        20overlayfs/overlayfs-mount.sh
-Patch0:         disable-xattr.patch
-Patch1:         fix-initrd-naming-for-mariner.patch
-Patch2:         fix-functions-Avoid-calling-grep-with-PCRE-P.patch
+URL:            https://github.com/dracutdevs/dracut/wiki
+
+Source0:        https://github.com/dracutdevs/dracut/archive/refs/tags/%{version}.tar.gz#/%{name}-%{version}.tar.gz
+Source1:        mkinitrd
+Source2:        megaraid.conf
+Source3:        20overlayfs/module-setup.sh
+Source4:        20overlayfs/overlayfs-mount.sh
+Patch:          fix-functions-Avoid-calling-grep-with-PCRE-P.patch
```

Comment:
Oh interesting - just commenting that it is interesting to see no patch numbering, but it does work as expected

```
time="2024-02-07T16:29:54Z" level=debug msg="Executing(%prep): /bin/sh -e /var/tmp/rpm-tmp.qBGOpv"
time="2024-02-07T16:29:54Z" level=debug msg="+ umask 022"
time="2024-02-07T16:29:54Z" level=debug msg="+ cd /usr/src/mariner/BUILD"
time="2024-02-07T16:29:54Z" level=debug msg="+ /usr/lib/rpm/mariner/gen-ld-script.sh dracut 059 /usr/src/mariner 3.0.20240207.1628"
time="2024-02-07T16:29:54Z" level=debug msg="gen-ld-script.sh name(dracut) version(059) _topdir(/usr/src/mariner) osversion(3.0.20240207.1628)"
time="2024-02-07T16:29:54Z" level=debug msg="==== ELF note generator v2.1.2 ===="
time="2024-02-07T16:29:54Z" level=debug msg="outdir is: /usr/src/mariner/BUILD/"
time="2024-02-07T16:29:54Z" level=debug msg="Stamp method: LinkerOnly"
time="2024-02-07T16:29:54Z" level=debug
time="2024-02-07T16:29:54Z" level=debug msg="#ifndef _AUTO_MODULE_INFO_H_"
time="2024-02-07T16:29:54Z" level=debug msg="#define _AUTO_MODULE_INFO_H_"
time="2024-02-07T16:29:54Z" level=debug
time="2024-02-07T16:29:54Z" level=debug msg="#define MODULE_VERSION      \"059.0.0.0\""
time="2024-02-07T16:29:54Z" level=debug msg="#define PACKAGE_VERSION     \"059\""
time="2024-02-07T16:29:54Z" level=debug msg="#define PACKAGE_NAME        \"dracut\""
time="2024-02-07T16:29:54Z" level=debug msg="#define TARGET_OS           \"mariner\""
time="2024-02-07T16:29:54Z" level=debug msg="#define TARGET_OS_VERSION   \"3.0\""
time="2024-02-07T16:29:54Z" level=debug
time="2024-02-07T16:29:54Z" level=debug msg="#endif //_AUTO_MODULE_INFO_H_"
time="2024-02-07T16:29:54Z" level=debug msg="Generated /usr/src/mariner/BUILD/auto_module_info.h file..."
time="2024-02-07T16:29:54Z" level=debug
time="2024-02-07T16:29:54Z" level=debug msg="Successfully generated module_info.ld and .note.package files..."
time="2024-02-07T16:29:54Z" level=debug msg="+ cd /usr/src/mariner/BUILD"
time="2024-02-07T16:29:54Z" level=debug msg="+ rm -rf dracut-059"
time="2024-02-07T16:29:54Z" level=debug msg="+ /usr/lib/rpm/rpmuncompress -x /usr/src/mariner/SOURCES/dracut-059.tar.gz"
time="2024-02-07T16:29:54Z" level=debug msg="+ STATUS=0"
time="2024-02-07T16:29:54Z" level=debug msg="+ '[' 0 -ne 0 ']'"
time="2024-02-07T16:29:54Z" level=debug msg="+ cd dracut-059"
time="2024-02-07T16:29:54Z" level=debug msg="+ /bin/chmod -Rf a+rX,u+w,g-w,o-w ."
time="2024-02-07T16:29:54Z" level=debug msg="+ /usr/lib/rpm/rpmuncompress /usr/src/mariner/SOURCES/fix-functions-Avoid-calling-grep-with-PCRE-P.patch"
time="2024-02-07T16:29:54Z" level=debug msg="+ /bin/patch -p1 -s --fuzz=0 --no-backup-if-mismatch -f"
time="2024-02-07T16:29:54Z" level=debug msg="+ /usr/lib/rpm/rpmuncompress /usr/src/mariner/SOURCES/allow-liveos-overlay-no-user-confirmation-prompt.patch"
time="2024-02-07T16:29:54Z" level=debug msg="+ /bin/patch -p1 -s --fuzz=0 --no-backup-if-mismatch -f"
time="2024-02-07T16:29:54Z" level=debug msg="+ /usr/lib/rpm/rpmuncompress /usr/src/mariner/SOURCES/0001-Add-mkinitrd-support-to-dracut.patch"
time="2024-02-07T16:29:54Z" level=debug msg="+ /bin/patch -p1 -s --fuzz=0 --no-backup-if-mismatch -f"
time="2024-02-07T16:29:54Z" level=debug msg="+ /usr/lib/rpm/rpmuncompress /usr/src/mariner/SOURCES/0002-disable-xattr.patch"
time="2024-02-07T16:29:54Z" level=debug msg="+ /bin/patch -p1 -s --fuzz=0 --no-backup-if-mismatch -f"
time="2024-02-07T16:29:54Z" level=debug msg="+ /usr/lib/rpm/rpmuncompress /usr/src/mariner/SOURCES/0003-fix-initrd-naming-for-photon.patch"
time="2024-02-07T16:29:54Z" level=debug msg="+ /bin/patch -p1 -s --fuzz=0 --no-backup-if-mismatch -f"
time="2024-02-07T16:29:54Z" level=debug msg="+ /usr/lib/rpm/rpmuncompress /usr/src/mariner/SOURCES/0004-fix-hostonly.patch"
time="2024-02-07T16:29:54Z" level=debug msg="+ /bin/patch -p1 -s --fuzz=0 --no-backup-if-mismatch -f"
time="2024-02-07T16:29:54Z" level=debug msg="+ /usr/lib/rpm/rpmuncompress /usr/src/mariner/SOURCES/0005-mkinitrd-verbose-fix.patch"
time="2024-02-07T16:29:54Z" level=debug msg="+ /bin/patch -p1 -s --fuzz=0 --no-backup-if-mismatch -f"
time="2024-02-07T16:29:54Z" level=debug msg="+ /usr/lib/rpm/rpmuncompress /usr/src/mariner/SOURCES/0006-dracut.sh-validate-instmods-calls.patch"
time="2024-02-07T16:29:54Z" level=debug msg="+ /bin/patch -p1 -s --fuzz=0 --no-backup-if-mismatch -f"
time="2024-02-07T16:29:54Z" level=debug msg="+ /usr/lib/rpm/rpmuncompress /usr/src/mariner/SOURCES/0007-feat-dracut.sh-support-multiple-config-dirs.patch"
time="2024-02-07T16:29:54Z" level=debug msg="+ /bin/patch -p1 -s --fuzz=0 --no-backup-if-mismatch -f"
time="2024-02-07T16:29:54Z" level=debug msg="+ /usr/lib/rpm/rpmuncompress /usr/src/mariner/SOURCES/0008-fix-dracut-systemd-rootfs-generator-cannot-write-out.patch"
time="2024-02-07T16:29:54Z" level=debug msg="+ /bin/patch -p1 -s --fuzz=0 --no-backup-if-mismatch -f"
time="2024-02-07T16:29:54Z" level=debug msg="+ /usr/lib/rpm/rpmuncompress /usr/src/mariner/SOURCES/0009-install-systemd-executor.patch"
time="2024-02-07T16:29:54Z" level=debug msg="+ /bin/patch -p1 -s --fuzz=0 --no-backup-if-mismatch -f"
time="2024-02-07T16:29:54Z" level=debug msg="+ RPM_EC=0"
time="2024-02-07T16:29:54Z" level=debug msg="++ jobs -p"
time="2024-02-07T16:29:54Z" level=debug msg="+ exit 0"
```